test.js

```diff
@@ -2,4 +2,11 @@
 
 // wrong codes
 var a = "1";
-a.includes("1");
+
+a.includes('1')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ipf-jypark/test-repo/pullRequest?prid=7372446)
```suggestion
a.includes("1")
```